test/ode/ode_dense_tests.jl

```diff
@@ -64,6 +64,24 @@ sol2 =solve(prob,Midpoint(),dt=1//2^(4),dense=true)
 
 prob = prob_ode_linear
 
+sol =solve(prob,SSPRK22(),dt=1//2^(2),dense=true)
+
+sol(interpd_1d,0:1//2^(4):1)
+
+sol2 =solve(prob,SSPRK22(),dt=1//2^(4),dense=true)
+
+@test maximum(map((x)->maximum(abs.(x)),sol2[:] - interpd_1d)) < 0.15
```

Comment:
Looks good. You'd know if the interpolation was bad because you'd see little bumps where the steps are. With the tolerance low enough, it'll be a regression test which makes sure no change causes that to happen in the future. 1.5e-2 is probably good enough considering what `Midpoint` is at. I'll wait for tests to pass and then merge.